### PR TITLE
help: clean up LogRecorder.logger

### PR DESCRIPTION
**Pull request by jsoref**
* the ui shows INFO
* typos (specified)
* grammar

### Proposed changelog entries

* help: clean up LogRecorder.logger

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

### CODE REVIEW

File: war/src/main/webapp/help/LogRecorder/logger.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <div>
     Specify a set of logger names and the corresponding log levels that
     this log recorder will collect. For example, if you specify
-    <tt>hudson.model.Hudson</tt> and <tt>info</tt>, any logs recorded
+    <tt>hudson.model.Hudson</tt> and <tt>INFO</tt>, any logs recorded
     for <tt>hudson.model.Hudson</tt> or its descendants with the log level
-    info or above will show up in this log recorder, unless a more specific
-    logger with higher log level is also specific.
+    INFO or above will show up in this log recorder unless a more specific
+    logger with higher log level is also specified.
```
Comment (daniel-beck) [inline war/src/main/webapp/help/LogRecorder/logger.html]: Bad example, as there can be no more specific logger than hudson.model.Hudson.

**Comment by jsoref** [comment]
@daniel-beck: updated

File: war/src/main/webapp/help/LogRecorder/logger.html [hunk 2]
```diff
@@ -1,13 +1,14 @@
 <div>
     Specify a set of logger names and the corresponding log levels that
     this log recorder will collect. For example, if you specify
-    <tt>hudson.model.Hudson</tt> and <tt>info</tt>, any logs recorded
-    for <tt>hudson.model.Hudson</tt> or its descendants with the log level
-    info or above will show up in this log recorder, unless a more specific
-    logger with higher log level is also specific.
+    <tt>hudson.model</tt> and <tt>INFO</tt>, any logs recorded
```
Comment (daniel-beck) [inline war/src/main/webapp/help/LogRecorder/logger.html]: Grammar?

**Review by daniel-beck — APPROVED** [review]
Looks good modulo some grammar I'm unsure about.

For bonus 🏅 , use Jenkins-themed examples (`jenkins.model.Jenkins` is a logger).

File: war/src/main/webapp/help/LogRecorder/logger.html (hunk 2 above)
Comment (jsoref) [reply war/src/main/webapp/help/LogRecorder/logger.html]: Please expound upon your concern here?

**Comment by jsoref** [comment]
@daniel-beck: updated

File: war/src/main/webapp/help/LogRecorder/logger.html (hunk 2 above)
Comment (daniel-beck) [reply war/src/main/webapp/help/LogRecorder/logger.html]: Bad reading of the unified diff.

**Review by daniel-beck — APPROVED** [review]

**Review by alecharp — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**